### PR TITLE
Backport main to net7.0

### PR DESCRIPTION
**Pull request by Redth**
Backport main to net7.0

### CODE REVIEW

File: GitInfo.txt [hunk 1]
```diff
@@ -1 +1 @@
-6.0.500-preview.1
+7.0.0-preview.1
```
Comment (jonathanpeppers) [inline GitInfo.txt:1]: Maybe doesn't need to happen here, but I already put Android as `-preview.6` because we're on .NET 7 Preview 6.

**Review by jonathanpeppers — APPROVED** [review]
Looks good 👍 , I would use a merge commit when you merge this (don't squash).

**Comment by mattleibow** [comment]
Having issues with the Public API checkers and one delegate in Tizen.

**Pull request merged**